apps/builder/app/builder/features/props-panel/props-panel.stories.tsx

```diff
@@ -40,6 +42,13 @@ pagesStore.set({
   ],
 });
 
+const urlSections = new Map([
+  [unique(), "links"],
+  [unique(), "about"],
+]);
+
+setMockUrlSectionsStore(atom(urlSections));
+
 const imageAsset = (name = "cat", format = "jpg"): Asset => ({
   id: unique(),
```

Comment:
wouldn't it be better to just use nanoid everywhere instead of unieque?

Reply:
Ah, just realised it will not work, because IDs are visible in lostpixel screenshot and have to be stable